uint32 car_id
---
bool outside_nav_flag